src/components/Slider/Slider.spec.tsx

```diff
@@ -0,0 +1,36 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { mount } from "@cypress/react";
+import { useState } from "react";
+import Slider from "./Slider";
+import css from "./Slider.module.css";
+
+const SLIDER_ITEMS = [
+    { id: "a", name: "abc" },
+    { id: "b", name: "def" },
+    { id: "c", name: "ghi" },
+];
+
+describe("Slider Component", () => {
+    it("should render foo text correctly", () => {
```

Comment:
```suggestion
    it("renders", () => {
```
I would move out the click to its own test `it('handles click and change the active item')`